components/organisms/DesktopMenu.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import styled from 'styled-components';
+import Link from 'next/link';
+
+import { IoChevronDownCircleOutline } from 'react-icons/io5';
+
+import { DefaultParagraph } from 'components/atoms/typography.styles';
+import { Green050 } from 'components/styling/colors';
+import StyledLink from 'components/atoms/StyledLink.styles';
+
+const DesktopMenuContainer = styled.div`
+  display: grid;
+  grid-template-columns: repeat(5, auto);
+  height: 28px;
+`;
+
+const DesktopMenuItem = styled(DefaultParagraph)`
+  height: 62px;
+  line-height: 62px;
+  align-self: center;
+`;
+
+const ChevronContainer = styled.span`
+  height: 62px;
+  position: relative;
+  top: 8px;
+  padding-left: 8px;
+  cursor: pointer;
+`;
+
+const DesktopMenu = () => (
+  <DesktopMenuContainer>
+    <DesktopMenuItem color={Green050}>
+      <Link href="/" passHref>
+        <StyledLink>
+          Magamról
+        </StyledLink>
+      </Link>
+    </DesktopMenuItem>
+    <DesktopMenuItem color={Green050}>
```

Comment:
Don't forget that without the `key` property, React won't know which DesktopMenuItem you would like to manipulate (you also get an error message for this in your browser's console).

Essentially imagine it like this. You have 5 apples on the table. One of them has a worm that you want to remove. However, you don't know which one has the worm. Obviously you start cutting apples until you find it, but at the end it's not like you have 4 completely fresh and nice apples, but you have X amount cut in half.

Now this happens if you don't provide keys for same level items in the virtual DOM. Just to keep with our example, imagine someone walking in the door and telling you that the 3rd apple needs to be cut, as that one has the worm. Obviously you cut it, then you still have 4 fresh apples, because you knew, which one to cut.

Reply:
nice analogy :D

Reply:
Also Balint just told me I forgot to state that this is only true in **loops**! I saw Miki suggesting it and defaulted to that. So just a disclaimer, you do not need keys outside of any loop as those elements are considered static 😄

Reply:
It was suspicious that I don't always see it in React code and that the others didn't mention it either :)